test/t8n/t8n.cpp

```diff
@@ -47,7 +47,7 @@ int main(int argc, const char* argv[])
 
         if (arg == "-v")
         {
-            std::cout << "evmone dev\n";  // FIXME: Return proper version.
+            std::cout << "evmone-t8n " << PROJECT_VERSION << std::endl;
```

Comment:
```suggestion
            std::cout << "evmone-t8n " PROJECT_VERSION "\n";
```

Reply:
Why is it better?

Reply:
The string will be concatenated by the compiler and `std::cout` called only once.

Reply:
Done